cmd/cli/install_test.go

```diff
@@ -25,6 +26,10 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/fake"
 
+	yaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/openservicemesh/osm/pkg/apis/config/v1alpha1"
+
```

Comment:
nit : no need of a new line here. All osm packages can come in the same block

Reply:
I suggested following this pattern where autogenerated packages are grouped separately. It is useful to catch dependency issues that might creep in from API code referring to other local packages. Let's keep this as-is for this reason.